### PR TITLE
[CLEANUP beta] #16391 Cleaning up the test output

### PR DESCRIPTION
**Pull request by KamiKillertO**
This resolves some Quest Tasks of Issue #16391:

- `Error: Catch me`
- `Error: the error.` 
- `Error: Element .does-not-exist not found.`

I've just remove a call to `console.error`, I don't know if he was here on purpose 🤷‍♂️.

### CODE REVIEW

File: packages/ember-testing/lib/test/adapter.js [hunk 1]
```diff
@@ -28,6 +28,4 @@ export function asyncEnd() {
 
 function adapterDispatch(error) {
   adapter.exception(error);
-  
-  console.error(error.stack);// eslint-disable-line no-console
```
Comment (rwjblue) [inline packages/ember-testing/lib/test/adapter.js]: Yep, we need to bring this back (and explicitly mock in tests where we expect an error).

Specifically, this console.error is the _only_ thing that notifies the console of failures (including the stack trace) when unexpected exceptions or rejections happen in tests...

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: packages/ember-testing/lib/test/adapter.js (hunk 1 above)
Comment (KamiKillertO) [reply packages/ember-testing/lib/test/adapter.js]: Something like this?

```javascript
function adapterDispatch(error) {
   adapter.exception(error);
   
   adapter.outputError(error.stack); 
```

`outputError` is by default `console.error` and can be overridden with a noop like this.

```
Test.adapter = QUnitAdapter.create({
   outputError () {}
});
```

Comment (rwjblue) [reply packages/ember-testing/lib/test/adapter.js]: I was more thinking to just override `console.error` in the specific tests, then reset it in the `afterEach`

**Review by rwjblue — APPROVED** [review]

**Comment by rwjblue** [comment]
Thank you!!

**Pull request merged**